doc/getting-started.md

```diff
@@ -595,6 +595,8 @@ find ods-configuration -name '*.sample' -type f | while read NAME ; do mv "${NAM
 
 Now you will have to check the `.env` configuration files in `ods-configuration`. Change all values with the suffix `_base64` to a Base64 encoded value.  
 
+If you want later changes to the repositories, update `ods-configuration-sample` and copy the configuration-sample.sh script located in ods-core/infrastructure-setup/scripts to the parent folder of `ods-configuration-sample` and `ods-configuration`.  
```

Comment:
Hmm. Do we really need to copy? Why not keep it in `ods-core`? And if we want to have a different location, then I'd suggest to symlink ...